### PR TITLE
Initial Android binding + demo

### PR DESCRIPTION
**Pull request by ErisMik**

### CODE REVIEW

**Comment by ErisMik** [comment]
> I get invalid argument error for simulator/device -> tests all pass except getFrameLength

You get Invalid Argument even just trying to run the demo?

**Comment by ksyeo1010** [comment]
> > I get invalid argument error for simulator/device -> tests all pass except getFrameLength
> 
> You get Invalid Argument even just trying to run the demo?

yes

**Comment by ErisMik** [comment]
> do we need to add libs to lib directory?

Yes ;)

**Review by ksyeo1010 — APPROVED** [review]

File: binding/android/cheetah/src/main/java/ai/picovoice/cheetah/Cheetah.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    }
+
+    private final long handle;
+
+    /**
+     * Constructor.
+     *
+     * @param accessKey AccessKey obtained from Picovoice Console
+     * @param modelPath Absolute path to the file containing Cheetah model parameters.
+     * @throws CheetahException if there is an error while initializing Cheetah.
```
Comment (kenarsa) [inline binding/android/cheetah/src/main/java/ai/picovoice/cheetah/Cheetah.java:43]: missing doc for `endpointDuration`

**Review by kenarsa — CHANGES_REQUESTED** [review]

**Pull request merged**